projects/core/src/lib/layers/measurement.ts

```diff
@@ -82,4 +83,9 @@ export class ClassificationMeasurement extends Measurement {
             classes: this.classes.toObject(),
         };
     }
+
+    colorHint(key: string): string | undefined {
```

Comment:
why do you call it color hint? It's not a color, so this may be confusing. Why not call it class label?

Reply:
because it is related to the color selection. So we might add a hint for min max values or something like that. It could be used for other colorizers then classifications.